keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -289,7 +367,7 @@ To raise awareness of annotation usage (in case of old automation), an additiona
 will be added when a pod is created with a seccomp annotation:
 
```

Comment:
> If _only_ seccomp annotations are specified, copy the values into the corresponding fields.
> This ensures that existing applications continue to enforce seccomp, and prevents the kubelet from
> needing to resolve annotations & fields.

It's unclear from this whether 1.19 kubelets will use annotations or pod fields. Because the annotation -> field copy is only done at pod creation time, and a pod scheduled against a 1.19 kubelet could have been created or updated against a 1.18 API server without the seccomp field, if 1.19 kubelets only look at pod fields, they could miss seccomp annotations that previously worked.

Reply:
I added a new section around this.